docs/application/dotnet/guides/tizen.circularUI/ConfirmPopupEffect.md

```diff
@@ -0,0 +1,107 @@
+---
+uid: Tizen.Wearable.CircularUI.doc.ConfirmPopupEffect
+summary: ConfirmPopupEffect guide
+---
+# ConfirmPopupEffect
+
+`ConfirmPopupEffect` is an [Effect](https://developer.xamarin.com/api/type/Xamarin.Forms.Effect/) for Xamarin.Forms that is used to create the [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage) with one or two buttons.
+A popup with only one button will have an Accept button only. The default text for Accept button is OK.
```

Comment:
A popup with only one button will have an Accept button only

If Accept is name of the button, then we need to bold the name.
Please change Accept to **Accept**